projects/plutus/inc/current_sense.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+// Current sense module for the LTC2484 ADC
+
+#include "ltc_adc.h"
+#include "status.h"
+
+typedef void (*CurrentSenseCallback)(int32_t current, void *context);
+
+typedef struct {
+  int32_t voltage;  // Voltage in microvolts
+  int32_t current;  // Voltage in milliamps
```

Comment:
I'm pretty sure the ADC has a higher ratio than mA. What raw value were you reading at 3A compared to 0A?

Reply:
Those comments are outdated. The units used by the module are actually defined by the current value that gets passed into `current_calibration_sample_point`, so arbitrary unit supported by the ADC can be used.

Reply:
That doesn't sound consistent. This module should use a fixed unit.

Reply:
Sorry, I meant that the math will work with any unit we use, so using a particular unit would just be a matter of passing in the max current into `current_calibration_sample_point` in the specific unit we want to use.

Reply:
That's still inconsistent? I don't want to have to recalibrate to know what unit I'm using.